postgre-client/src/asset_index_client.rs

```diff
@@ -260,17 +270,31 @@ impl AssetIndexStorage for PgClient {
             assets_table: "assets_v3".to_string(),
             creators_table: "asset_creators_v3".to_string(),
             authorities_table: "assets_authorities".to_string(),
-            fungible_tokens_table: "fungible_tokens".to_string(),
         };
-        self.upsert_batched(&mut transaction, table_names, updated_components)
+        self.upsert_batched_non_fungible(&mut transaction, table_names, updated_components)
             .await?;
         self.commit_transaction(transaction).await
     }
 
+    async fn update_fungible_asset_indexes_batch(
+        &self,
+        fungible_asset_indexes: &[FungibleAssetIndex],
+    ) -> Result<(), IndexDbError> {
+        let mut transaction = self.start_transaction().await?;
```

Comment:
please use to start_commit a similar approach to this change: https://github.com/metaplex-foundation/aura/pull/310/commits/8217fab33a41805e458a1c52625682db5f1daa0f